inventory.py

```diff
@@ -14,6 +14,7 @@ def parse_params():
     parser.add_option("-d", "--debug", help="enable debug", action="store_true", dest="debug", default=False)
     parser.add_option("-l", "--loopback", help="use 127.0.0.1 address instead of listen to 0.0.0.0",
                       action="store_true", dest="loopback")
+    parser.add_option("-s", "--sleep", help="specifiy sleep time for inventory builder", action="store", dest="sleep")
```

Comment:
would indicate its optional character, mentioning the default value

Reply:
Ok